src/ncp/ncp_base.cpp

```diff
@@ -55,6 +55,10 @@
 #include "openthread/jam_detection.h"
 #endif
 
+#if OPENTHREAD_ENABLE_RAW_LINK_API
+#include "openthread/link_raw.h"
+#endif
```

Comment:
I don't think this should be required, as NCP code is already referencing all the other LinkRaw APIs just fine.